script.js

```diff
@@ -1,27 +1,53 @@
 // an array with all of our cart items
 var cart = [];
+var total = 0;
 
 var updateCart = function () {
-  // TODO: finish
-}
 
+	var source = $('#cart-template').html();
+
+	// compile our template html using handlebars
+	var template = Handlebars.compile(source);
+
+	// template can only take an object; reference in HTML with "cart"
+	var newHTML = template({cart});
+
+	// since using {{#each}}, replace the html each time
+	$('.cart-list').html(newHTML);
+
+	$('.total').html(total);
```

Comment:
i think mine was set to indent 4, and yours to 2, and i ad-hoc fixed it, but didn't consolidate all the indentation. 

life's tough sometimes man